### PR TITLE
Dh test coverage

### PR DESCRIPTION
**Pull request by DavidHoltkamp1**
# Description :: User Story

- Increase test coverage, mainly in our upvotes controller. 
- Probably can explore a little further edge cases if we feel the need. 
- Would like to test that you cannot click on the arrow 'button' after you have done so. 
## Type of change

- [ ] New Feature
- [ ] Bug Fix
- [x] Refactor
- [ ] Breaking Change

## RSpec results

```
Finished in 9.76 seconds (files took 3.09 seconds to load)
93 examples, 0 failures

Coverage report generated for RSpec to /Users/davidholtkamp521/turing/3module/projects/Turing.SO/coverage. 293 / 330 LOC (88.79%) covered.
```

## Rubocop results

```
28 files inspected, 35 offenses detected
```

### CODE REVIEW

File: spec/features/upvotes/upvote_spec.rb [hunk 1]
```diff
@@ -16,6 +19,29 @@
...
+    @answer_1 = @question_1.answers.create!(
+      content: "A way to give access to methods outside the class",
+      user_id: @user_2.id
+    )
+
+    @answer_comment = @answer_1.comments.create(
+      content: "This answer worked for me",
+      user_id: @user_3.id
+    )
+
```
Comment (alerrian) [inline spec/features/upvotes/upvote_spec.rb:44]: Great job figuring out how to target all of this in a view that was huge.
